FutureNHS/futurenhs-platform
# Publish CreateWorkspace event

[AB#2137](https://dev.azure.com/futurenhs/75407963-f812-43e5-a734-1059bbc036a3/_workitems/edit/2137)

- Backfill CreateWorkspace event

## Pre-review checklist:

- [x] Analytics (user analytics, e.g. Google Analytics)
- [x] Observability (metrics/tracing)

## Testing information

- Create a workspace
- Create a subscriber for EventGrid to see if events have been published

## Test

- [ ] Tested in cluster
